content/automate/ManageIQ/Cloud/VM/Provisioning/StateMachines/VMProvision_VM.class/__methods__/retry_placement.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Description: This method updates the provisioning status and retries
+# from the Placement step.
+# Required inputs: status
+#
+
+prov = $evm.root['miq_provision']
+unless prov
+  $evm.log(:error, "miq_provision object not provided")
+  exit(MIQ_STOP)
+end
+status = $evm.inputs['status']
```

Comment:
Max retry is a per-step check so it will not work in this case.  You would need to use `set_state_var` and `get_state_var` to monitor the number of attempts.  https://github.com/pemcg/manageiq-automation-howto-guide/blob/master/chapter12/state_machines.md#saving-variables-between-state-retries